### ISSUE TITLE
can we get a build or instructions for an arm64 system?

### ISSUE THREAD

**Issue by SpyrosTheBoss** [fmang/oshu#96]
plz im begging you

**Comment by Eragonfr** [comment]
Can't you build it on the aarch64 host ?
If you want to cross-build it you should simply look at how cmake cross-compilation works. With an aarch64 toolchain and libraries installed you will be able to build it from your x86_64 host for your aarch64 target.
There is plenty of documentation about that on the web, just google it.

**Comment by fmang** [comment]
Indeed, building on the host would be the simplest option. The build instructions in the README should work on any architecture.
